### DISCUSSION TITLE
Putting HTML in JSON

### DISCUSSION THREAD

**Discussion by tiqvahub** [TerriaJS/terriajs#6711] [Q&A]
Hi developers and users,

I'm trying to put an [image](https://github.com/TerriaJS/TerriaMap/blob/main/wwwroot/config.json#L49) HTML element into the [extraCreditLinks](https://github.com/TerriaJS/terriajs/blob/main/lib/Models/Terria.ts#L533-L540) array in the "text" object property, but it seems not work. The HTML element is always shown as a text with the link stored in the "url" object property. Is there any other variable / object to add an extraCredit using a imageLogo with a link rather than a single text?

Julierme

**Comment by Jul-Git** [comment]
Hi,

If it's not solved already, you may try single quotes instead of double quotes to embed your image path:
`"description": "some text, etc <br><img src='some_path' alt='some_description' widht='100%' height='100%'><br> some other text if you need"`
